modules/ROOT/pages/amazon/amazon-sqs-connector-studio.adoc

```diff
@@ -0,0 +1,90 @@
+= Amazon SQS Studio Configuration - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+To configure a connector in Anypoint Studio:
+
+. Install the connector.
+. Configure an input source for the connector.
+. Configure the connector.
+
+== Install the Connector
+
+Installing the connector adds it to a Studio project. Anypoint Studio provides two ways to install a connector:
+
+* Use the Exchange button in the Studio taskbar.
+
+* Use the Exchange button in the Mule Palette.
+
+=== Install the Connector From the Studio Taskbar
+
+. In Studio, create a Mule project.
+. Click the Exchange *(X)* icon in the upper left of the Studio task bar.
+. In Exchange, click *Login* and supply your Anypoint Platform username and password.
+. In Exchange, search for `Amazon SQS`.
+. Select *Amazon SQS*.
+. Click *Add to project*.
+. Follow the prompts to install the connector.
+
+=== Install the Connector from the Mule Palette
+
+. In Studio, create a Mule project.
+. In the Mule Palette, click *(X) Search in Exchange*.
+. In *Add Modules to Project*, type the `Amazon SQS` in the search field.
+. Click the connector name in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+== Configure an Input Source
+
+Configure an input source for the connector such as a connector operation, *HTTP Listener*, or *Scheduler*.
+
+== Configure the Connector
+
+To use Amazon SQS Connector in your Mule application, configure a global element that can be used by all Amazon SQS Connectors in the application.
+
+. Drag a connector operation onto the Studio canvas.
+. To create a global element for the connector, click the *Global Elements* tab at the base of the canvas, and then click *Create*.
+. In the *Choose Global Type* window, expand *Connector Configuration*, and select *Amazon SQS Configuration*:
++
+image::amazon/amazon-sqs-studio-select-global.png[Select Connector for Global Element]
++
+. Click *OK*.
+. Specify the global element properties.
++
+You can either enter your credentials into the global configuration properties or reference a configuration placed in a file in the `src` folder of your project file that contains these values. For simpler maintenance and better reuse of your project, use a configuration file.
++
+For example:
```

Comment:
End the first para with a colon and omit "For example:".

Whenever possible, I'd like to avoid the fragment "For example:".